frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -80,10 +81,18 @@ const validateUserTemplate: VmSettingsValidator = (field, options) => {
   if (!userTemplateName) {
     return null;
   }
+
   const userTemplate = iGetLoadedCommonData(state, id, VMWizardProps.userTemplates, List()).find(
     (template) => iGetName(template) === userTemplateName,
   );
 
+  if (!userTemplate) {
+    return asValidationObject(
+      'Template is missing, check template name',
```

Comment:
maybe we should ask somebody from UX to have a better message :)

it is not that the template name is wrong - it is just the template is missing for unknown reason (probably got deleted right now)

Reply:
change to be less instructive :-)